logstash-core/spec/logstash/util/wrapped_synchronous_queue_spec.rb

```diff
@@ -45,7 +45,7 @@ def take() shift(); end
       def poll(*) shift(); end
     end
 
-    describe "WriteClient | ReadClient" do
+    xdescribe "WriteClient | ReadClient" do
```

Comment:
@jordansissel these are broken because the `DummyQueue` here doesn't expose the `queue` accessor. I think these are irrelevant now, but they also test some stuff about the metrics. Let me double check if these are actually redundant before killing them.